### PR TITLE
Add vsp renderer on weston 5

### PR DESCRIPTION
**Pull request by rshym**

### CODE REVIEW

**Comment by rshym** [comment]
> Looks like it depends on device-tree configuration. I.e. if there a "name" node is present

Thanks for pointing.I was thinking that this is some change inside new weston or something. But is this depends on

**Comment by rshym** [comment]
> Looks like it depends on device-tree configuration. I.e. if there a "name" node is present

Thanks for pointing.
I was thinking that this is some change inside new weston or something.
But is this depends on device tree, it's better to have both <device_module>-device and <bus_info>-device.

**Comment by otyshchenko1** [comment]
Sometime we had changes to weston.ini to inform Weston which VSP to use.  Why we don't need them now?
[v4l2-renderer]
device=/dev/media1
device-module=vsp2

**Comment by rshym** [comment]
> Sometime we had changes to weston.ini to inform Weston which VSP to use. Why we don't need them now?
> [v4l2-renderer]
> device=/dev/media1
> device-module=vsp2

Thanks! I will investigate this.

**Comment by otyshchenko1** [comment]
> > Sometime we had changes to weston.ini to inform Weston which VSP to use. Why we don't need them now?
> > [v4l2-renderer]
> > device=/dev/media1
> > device-module=vsp2
> 
> Thanks! I will investigate this.

Please see:
https://github.com/renesas-rcar/meta-renesas/commit/e46d6a91dc9e8f0383aa604878ec2d8db04a85ce#diff-734c029fe26558ab394fcc34cbc19d6c

Also please note that different VSP should be used for h3ulcb/m3ulcb:
https://github.com/renesas-rcar/meta-renesas/commit/e46d6a91dc9e8f0383aa604878ec2d8db04a85ce#diff-b7dfb34a03a43b131f813fd05382ca77L32

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston [hunk 1]
```diff
@@ -287,7 +287,7 @@ LICENSE_FLAGS_WHITELIST = "commercial"
 DISTRO_FEATURES_append = " mm-test"
 
 # for weston v4l2 renderer
-#DISTRO_FEATURES_append = " v4l2-renderer"
+DISTRO_FEATURES_append = " v4l2-renderer"
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston]: As we agreed this shouldn't be enabled for all machines, but for H3 ES3.0 based board only.

Comment (rshym) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston]: I will fix.

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston [hunk 2]
```diff
@@ -411,7 +412,6 @@ DISTRO_FEATURES_append = " virtualization"
 DISTRO_FEATURES_remove = " x11 gtk gobject-introspection-data nfc irda zeroconf 3g"
 
 # Remove multimedia options if XT_RCAR_PROPRIETARY_MULTIMEDIA_DIR is not defined
-MACHINE_FEATURES_remove = " ${@bb.utils.contains('XT_RCAR_PROPRIETARY_MULTIMEDIA_DIR','','','multimedia', d)}"
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston]: If neither v4l2-renderer or XT_RCAR_PROPRIETARY_MULTIMEDIA_DIR are in use, we will have multimedia present in MACHINE_FEATURES, is it ok? Other changes LGTM

Comment (rshym) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston]: Good point, thanks!I reworked this part, and addressed my own concern regarding not clear dependency between vsp-renderer and multimedia. So I made "backward" dependency: 'multimedia' machine feature is enabled only if it is required by use case. Like multimedia package is provided or vsp-renderer is enabled.

Comment (rshym) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston]: Also I rewrote commit message to describe this change. And add appropriate comment to place where this machine feature is enabled previously.

Comment (rshym) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston]: Wrong decision. Will be rewritten.

Comment (rshym) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston]: For now this block is returned to state as mentioned in first comment - just remove line that removes 'multimedia'.

**Pull request merged**